python/cuml/solvers/qn.pyx

```diff
@@ -214,10 +214,10 @@ class QN(Base):
 
     def __init__(self, loss='sigmoid', fit_intercept=True,
                  l1_strength=0.0, l2_strength=0.0, max_iter=1000, tol=1e-3,
-                 linesearch_max_iter=50, lbfgs_memory=5, verbose=False,
+                 linesearch_max_iter=500, lbfgs_memory=5, verbose=0,
```

Comment:
Wow, that's a lot of line search steps! scipy defaults to 20. Any thoughts on why we need so many line seaches?

Reply:
I was waaaay too overzealous, I never tried leaving the iterations after correcting the C++ loss side, but it actually works with 20 and 50 so will revert this change back. We sometimes need a bit more than the default scipy due to some small differences in the code in my experience, but 50 should be more than enough